bpdv3/Assets/Scripts/Player/PlayerDestination.cs

```diff
@@ -42,7 +42,8 @@ private void OnTriggerEnter(Collider other)
                 if (!IsDestinationObtained)
                 {
                     _playerMovement.nextMoveTileGO = other.gameObject;
-                    _playerMovement.tilePropertiesOnNextMoveTileGO = other.gameObject.GetComponentInParent<TileProperties>();
```

Comment:
I think my IDE did this spacing change by itself. It was not intentional. It just moved how it reads, onto 2 lines instead of 1. Nothing was changed here.